review-responses.tex

```diff
@@ -0,0 +1,383 @@
+    proficient in coalescent theory. Therefore, please improve the introduction by
+    providing more information on: \emph{(points below)}
+\end{point}
+
+\reply{
+    Thanks for the suggestion;
+    since this paper is entirely about forwards simulation,
+    we hope not to depend on coalescent theory at all in any crucial way.
+    We have added some more explanation of the relationship to simplification
+    at \revref;
+    and have tried to better signpost the technical arguments at \llname{ll:coal_signpost}.
+}
+
+
+\begin{point}{}
+    The idea of running simulations forward in time for the demography and then
+    backward in time for the neutral genetics is by far not new and was already
+    implemented in the software SPLATCHE (Ray, Currat, Foll, Excoffier,
+    Bioinformatics 2010, 26: 2993-2994), although probably not as efficiently. The
+    reference should be added.
+\end{point}
+
+\reply{
+    Thanks for drawing our attention to the (good) analogy 
+    between the forward-backward scheme used by SPLATCHE
+    and the process of tree sequence simplification,
+    which we had not considered.
+    While there are indeed deep connections between this approach and tree sequence simplification, 
+    they are however fundamentally different:
+    the coalescent is a stochastic process that is dual to the forwards-time process,
+    and relies on certain assumptions for correctness;
+    whereas tree sequence simplification is deterministic (just ``bookkeeping'').
+    We have updated the introduction to discuss this connection with SPLATCHE \revref,
+    and have discussed further the relationship of coalescence to simplification \revreffull{1}{2}.
+}
+
+
+\begin{point}{}
+  What is the use of gene trees? What can be inferred/calculated from them that
+  we cannot get from the endpoint genetic polymorphism data? (An example comes
+very late in the ms, should be moved up.)
+\end{point}
+
+\reply{
+    We agree that discussion of uses of gene trees would make nice motivation,
+    but think it is somewhat outside of the scope of this paper:
+    for our purposes here, it suffices to explain that they help everything go faster.
+    We've expanded the discussion \revref.
+}
+
+\begin{point}{}
+  Spell out once what is meant with $\theta = 4N_e\mu$ (and make clear that $\mu$ is
+  the num loci $\times$ mutation rate)
+\end{point}
+
+\reply{
+    We agree this was out of place; but rather than explain, we have removed this. \revref
+}
+
+
+\begin{point}{}
+  Make clear how to transfer from haploid to diploid case (I guess by having
+  about twice the data recorded).
+\end{point}
+
+\reply{
+    We've tried to make this more clear. \revref
+}
+
+\begin{point}{}
+  Figure 3 doesn't show the red stars mentioned in the main text (p8).
+\end{point}
+
+\reply{
+    These are visible for us; there may have been a problem with PDF transparency;
+    we've set opacity on these to 100\%; let us know if they are still invisible?
```

Comment:
I had an issue with this recently.  I found that certain inkscape objects with transparency did not display on OS X's Preview, but 'just worked' in Adobe Reader on the same platform (and all Linux pdf viewers that I tested)   Could be worth mentioning?

Reply:
Hm, maybe, but it shouldn't be a problem any more since they're not transparent?  I'm inclined to not mention. (also: grumph.)